### ISSUE TITLE
If there is an image with empty alt and text in a link, the aria-label gets added without text

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-new-window-warnings#15]
Example:
![image](https://github.com/user-attachments/assets/0a68c510-fd58-46bb-80e0-c9323d744e3d)

We need a way to account for the presence of both an image and text in a link.

**Comment by kishanjasani** [comment]
@amberhinds What should be `aria-label` if text and alt in image is not present?

**Comment by amberhinds** [comment]
I think that if an image is present and text, the text should probably always take precedence, because in many cases, the image alt is descriptive of the image and not link purpose, where as the visible text is more likely to be descriptive of link purpose.

With that in mind, I'm inclined to say that we always disregard image alt if there is any other text within the link. We should only pass the alt to the aria-label if there is nothing else inside the link.

**Comment by kishanjasani** [comment]
Got it. Thanks
